### PR TITLE
LP-2378 Pre-fill Business Line selected during Signup

### PR DESCRIPTION
**Pull request by annas-usman**
Business Line was being selected during sign-up hence when user opened cover letter, it should had been pre-filled.

[Ticket](https://philanthropyu.atlassian.net/browse/LP-2378)
[adg-edx-theme](https://github.com/OmnipreneurshipAcademy/adg-edx-theme/pull/93)

### CODE REVIEW

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/863/workflows/workflows_id/jobs/863/artifacts)**. Coverage !!! :scream:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/applications/views.py (50.0%): Missing lines 311-312
-------------
Total:   4 lines
Missing: 2 lines
Coverage: 50%
-------------
```

File: openedx/adg/lms/applications/views.py [hunk 1]
```diff
@@ -306,6 +306,11 @@ def get(self, request):
         business_lines = BusinessLine.objects.all()
         file_name = None
 
+        try:
+            registration_business_line = ExtendedUserProfile.objects.get(user=request.user).company
```
Comment (MunamMubashir-arbisoft) [inline openedx/adg/lms/applications/views.py:310]: We can get this as `user.extended_profile.company`

**Review by MunamMubashir-arbisoft — APPROVED** [review]
Added a comment, otherwise everything is fine.

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/864/workflows/workflows_id/jobs/864/artifacts)**. Coverage !!! :scream:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/lms/applications/views.py (50.0%): Missing lines 311-312
-------------
Total:   4 lines
Missing: 2 lines
Coverage: 50%
-------------
```

**Review by harris-irfan — APPROVED** [review]

**Pull request merged**